### ISSUE TITLE
Test failure

### ISSUE THREAD

**Issue by maelle** [timtrice/hurricanes#126]
:wave: @timtrice!

We see an error https://github.com/r-universe/ropensci/runs/5062182423?check_suite_focus=true via https://ropensci.r-universe.dev/ui#builds

```
* checking tests ...
  Running 'testthat.R'
 ERROR
Running the tests in 'tests/testthat.R' failed.
Last 13 lines of output:
   42.   \-rlang::cnd_signal(...)
  Warning messages:
  1: The `x` argument of `as_tibble.matrix()` must have unique column names if `.name_repair` is omitted as of tibble 2.0.0.
  Using compatibility `.name_repair`.
  This warning is displayed once every 8 hours.
  Call `lifecycle::last_lifecycle_warnings()` to see where this warning was generated. 
  2: `cols` is now required when using unnest().
  Please use `cols = c(Forecasts)` 
  3: `cols` is now required when using unnest().
  Please use `cols = c(Seas, WindRadius, Forecast)` 
  4: `cols` is now required when using unnest().
  Please use `cols = c(Forecasts)` 
  5: `cols` is now required when using unnest().
  Please use `cols = c(Seas, WindRadius, Forecast)` 
  Execution halted
```

Happy to help if needed!

**Comment by maelle** [comment]
Noting I sent an email.

**Comment by SimonAytes** [comment]
Closed by pull request #142.
